packages/ember-glimmer/lib/environment.js

```diff
@@ -79,14 +79,20 @@ export default class Environment extends GlimmerEnvironment {
     return false;
   }
 
-  getComponentDefinition(name) {
+  getComponentDefinition(nameSegments, meta) {
+    let name = nameSegments.join('/');
     let definition = this._components[name];
 
     if (!definition) {
-      let { component: ComponentClass, layout } = lookupComponent(this.owner, name[0]);
+      let { component: ComponentClass, layout } = lookupComponent(this.owner, name);
```

Comment:
Ahh, I think I need to get rid of the code you commented on below and instead use this third parameter.